tests/unit/test_charm.py

```diff
@@ -33,19 +33,19 @@ def setUp(self, network_mock):
 
     @mock.patch("charms.operator_libs_linux.v2.snap.Snap.start")
     def test_pollen_start(self, start_mock):
-        PollenService.start()
+        pollen = PollenService()
```

Comment:
The service can be instantiated in the constructor and store in a class attribute